### PR TITLE
Added TTL 74240, 74241, 74244

### PR DESCRIPTION
**Pull request by btb**
split from #1461

still to be abstracted

### CODE REVIEW

**Comment by maehne** [comment]
@btb: If you plan to refactor the implementation, please mark this PR as draft!

File: src/main/java/com/cburch/logisim/std/ttl/Ttl74241.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ *
+ * https://github.com/logisim-evolution/
+ *
+ * This is free software released under GNU GPLv3 license
+ */
+
+package com.cburch.logisim.std.ttl;
+
+/**
+ * TTL 74x241 octal buffers and line drivers with three-state outputs
```
Comment (maehne) [inline src/main/java/com/cburch/logisim/std/ttl/Ttl74241.java:13]: Why don't you use the more detailed description you added to `std.properties`?

File: src/main/java/com/cburch/logisim/std/ttl/Ttl74244.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ *
+ * https://github.com/logisim-evolution/
+ *
+ * This is free software released under GNU GPLv3 license
+ */
+
+package com.cburch.logisim.std.ttl;
+
+/**
+ * TTL 74x244 octal buffers and line drivers with three-state outputs
```
Comment (maehne) [inline src/main/java/com/cburch/logisim/std/ttl/Ttl74244.java:13]: Why don't you use the more detailed description you added to `std.properties`?

File: src/main/java/com/cburch/logisim/std/ttl/Ttl74240.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ *
+ * https://github.com/logisim-evolution/
+ *
+ * This is free software released under GNU GPLv3 license
+ */
+
+package com.cburch.logisim.std.ttl;
+
+/**
+ * TTL 74x240 octal buffers and line drivers with three-state outputs
```
Comment (maehne) [inline src/main/java/com/cburch/logisim/std/ttl/Ttl74240.java:13]: Why don't you use the more detailed description you added to `std.properties`?

**Review by maehne — CHANGES_REQUESTED** [review]
Looks mostly good to me! Thanks @btb!

File: src/main/java/com/cburch/logisim/std/ttl/Ttl74244.java (hunk 2 above)
Comment (btb) [reply src/main/java/com/cburch/logisim/std/ttl/Ttl74244.java:13]: this is basically the plain vanilla version, so the description is the same

File: src/main/java/com/cburch/logisim/std/ttl/Ttl74241.java (hunk 1 above)
Comment (btb) [reply src/main/java/com/cburch/logisim/std/ttl/Ttl74241.java]: fixed

File: src/main/java/com/cburch/logisim/std/ttl/Ttl74240.java (hunk 3 above)
Comment (btb) [reply src/main/java/com/cburch/logisim/std/ttl/Ttl74240.java]: fixed

**Review by MarcinOrlowski — CHANGES_REQUESTED** [review]
Please address linter complaints.

**Review by maehne — APPROVED** [review]
Except for the issue raised by @MarcinOrlowski. this looks good to me. Thanks!

**Review by maehne — APPROVED** [review]
@MarcinOrlowski: What needs to be still addressed to get this PR merged?

**Comment by BFH-ktt1** [comment]
@MarcinOrlowski : What is still blocking this PR?

**Pull request merged**